src/components/molecules/ScheduleEvent/ScheduleEvent.scss

```diff
@@ -83,9 +122,64 @@ $description--margin-top: 2px;
     justify-content: center;
     align-items: center;
     padding: $bookmark--padding;
+    margin-left: auto;
 
     &:hover {
       padding-top: $bookmark-hover--padding-top;
     }
+
+    &--hidden {
+      display: none;
+    }
+  }
+
+  &__expand {
+    color: $concrete--dark;
+    outline: none;
```

Comment:
I believe we shouldn't do that (i can't see the outline as the icon is not clickable)
```suggestion
```

Reply:
still need it tho